x-pack/plugins/aiops/public/components/document_count_content/document_count_chart/brush_badge.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+
+import { EuiBadge, EuiText, EuiToolTip } from '@elastic/eui';
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
+
+const DATE_FORMAT = 'YYYY-MM-DD';
+const TIME_FORMAT = 'HH:mm:ss';
```

Comment:
there is a Kibana plugin `fieldFormats` that we should use, it takes into account Kibana settings

Reply:
to avid an extra refactoring in a bugfix PR, decided to postpone this change with the `fieldFormats` plugin for 8.5

Reply:
Added an item to the `8.5` meta issue: https://github.com/elastic/kibana/issues/138117